common/artifacts/000_wc_artifacts.txt

```diff
@@ -838,6 +1212,227 @@ prosthetic_blade_hand = {
 
 	picture = "GFX_inv_weapon_shortblade_29"
 }
+warsong_battle_drums = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes
+		
+	}
+
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+	command_modifier = {
+#		actual_heavy_infantry = 0.05
+#		actual_light_infantry = 0.05
+#		archers = 0.05
+#		light_cavalry = 0.05
+		offence = 0.1
+		morale_offence = 0.1
+	}
+	picture = "GFX_warsong_battle_drums"
+	
+	slot = leader
+}
+scourge_bone_chimes = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes	
+			OR = {
+			is_death_knight_class_trigger = yes
+			is_necromancer_class_trigger = yes
+			is_warlock_class_trigger = yes
+			OR = {
+				religion = death_god
+				religion_group = fel_group
+			}
+		}
+	}
+
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+
+	general_opinion = -5
+	death_god_opinion = 5
+	command_modifier = {
+		morale_defence = 0.1
+		morale_offence = 0.1
+	}
+	picture = "GFX_scourge_bone_chimes"
+	
+	slot = leader
+}
+frost_wyrm_skull_shield = {
+	quality = 3
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes
+	}
+
+	indestructible = yes
+
+	martial = 2
+
+	combat_rating = 20
+
+	monthly_character_prestige = 0.75
+	
+	command_modifier = {
+		morale_defence = 0.05
+		defence = 0.05
+	}
+
+	picture = "GFX_frost_wyrm_skull_shield"
+	slot = libram
+}
+orb_of_kiljaeden = {
+	quality = 3
+	flags = { misc fel_artifact powerful_magical_artifact }
+
+	active = {
+		is_adult = yes
+		
+		use_dark_artifacts_trigger = yes
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 1
+	intrigue = 2
+
+	combat_rating = 15
+
+	monthly_character_prestige = 0.45
+	monthly_character_piety = 0.15
+	
+	general_opinion = -10
+	fel_group_opinion = 15
+
+	picture = "GFX_orb_of_kiljaeden"
+	slot = libram
+}
+orb_of_corruption = {
+	quality = 3
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes
```

Comment:
Why `orb_of_kiljaeden` has `use_dark_artifacts_trigger = yes`, but this artifact hasn't?

Reply:
done. but you realise, orb of kiljaeden didn't have it before last review, right?

Reply:
Whachu mean? `orb_of_kiljaeden` was added with `use_dark_artifacts_trigger = yes` trigger.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/67507647-69b23e00-f6a0-11e9-8e9b-a7c6c9d8e4b4.png)

</details>